packages/ag-charts-types/src/chart/chartOptions.ts

```diff
@@ -146,7 +146,7 @@ export interface AgKeyboardOptions {
      *
      * Default: `0`
      */
-    tabIndex?: number;
+    tabIndex?: 0 | -1;
```

Comment:
We can't change this in a minor release.

Reply:
Reverted here: https://github.com/ag-grid/ag-charts/commit/6332783884ddc04b971008ec7f6bd929dfe2a16a